src/clients/c/tb_client.zig

```diff
@@ -124,17 +124,20 @@ pub const Logging = struct {
 
 pub fn register_log_callback(
     callback_maybe: ?Logging.Callback,
+    debug: bool,
```

Comment:
:thinking: I don't love bare boolean arguments, but since this is exposed via bindings I guess it is simpler than adding a new options struct.

Reply:
Yeah - I'd like to keep the extern fn signatures as simple as possible! No structs (or u128s...).